src/style.css

```diff
@@ -0,0 +1,87 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
```

Comment:
well, it's not just simply removing `flex-grow`. you have hardcoded `flex-basis` to 300px. it will fail like this. but you can use percents

Reply:
think about the behavior that you want and how you can achieve it